tests/unit/test_charm.py

```diff
@@ -1,301 +1,513 @@
-# Copyright 2020 Canonical Ltd.
+# Copyright 2022 Canonical Ltd.
 # Licensed under the GPLv3, see LICENCE file for details.
 
-from mock import MagicMock, patch
-import unittest
+from unittest import mock
+import logging
+import os
 
-from ops.model import ActiveStatus, BlockedStatus, MaintenanceStatus
-from ops.testing import Harness
+import pytest
+from ops import model
+from ops import testing
```

Comment:
Same comment as above about explicit imports rather than importing "model" or "testing" for instance.

Reply:
Let's discuss it there and I'll adjust whatever outcome we reach there here as well

Reply:
Reverted it back